### PR TITLE
hive: migrate to openjdk formula

### PR DESCRIPTION
**Pull request by jonchang**
Mass migration of formulae that don't need bottles and depend on Java 1.8. Hopefully this works.

Part of https://github.com/Homebrew/homebrew-core/issues/63290.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jonchang** [comment]
Cherry picked non failing tests onto `master`. Will investigate test failures later.

File: Formula/jooby-bootstrap.rb
Comment (SMillerDev) [inline Formula/jooby-bootstrap.rb:4]: Last commit in 2014, open PR from 2015. I'd say remove it.

File: Formula/cassandra-reaper.rb
Comment (SMillerDev) [inline Formula/cassandra-reaper.rb:11]: Needs to point to a different port and log directory.

File: Formula/apache-flink.rb
Comment (SMillerDev) [inline Formula/apache-flink.rb:19]: Are we setting Java_home correctly here?

**Review by SMillerDev — COMMENTED** [review]
Suggestions based on the errors, I'm fine with merging as is and making an error tracking issue.

**Comment by Moisan** [comment]
I opened a PR to migrate `apache-flink` to `openjdk@11`: https://github.com/Homebrew/homebrew-core/pull/64818

**Review by SMillerDev — COMMENTED** [review]
LGTM now, just the audits remaining. I'll leave this to another maintainer to approve since I did the last changes

**Review by SMillerDev — DISMISSED** [review]

**Comment by jonchang** [comment]
Analytics for these last three formulae:

druid: stable 0.12.3
install: 15 (30 days), 42 (90 days), 251 (365 days)
install-on-request: 15 (30 days), 42 (90 days), 247 (365 days)
build-error: 0 (30 days)

hive: stable 3.1.2
install: 440 (30 days), 1,305 (90 days), 7,160 (365 days)
install-on-request: 415 (30 days), 1,220 (90 days), 6,656 (365 days)
build-error: 0 (30 days)

jooby-bootstrap: stable 0.2.2
install: 10 (30 days), 13 (90 days), 27 (365 days)
install-on-request: 10 (30 days), 13 (90 days), 27 (365 days)
build-error: 0 (30 days)

**Comment by jonchang** [comment]
Moving druid and jooby-boostrap over to https://github.com/Homebrew/homebrew-core/pull/64662 so there's one centralized issue to refer to for these java 1.8 requirement deprecations. This pull request will just merge `hive`.

**Pull request merged**
